discourse_migration/userguide.md

```diff
@@ -45,48 +45,49 @@ are the typical use cases:
 | All-commits           | no migration at the moment |
 | Bugs-admin            | no migration at the moment |
 | cfe-commits           | no migration at the moment |
-| cfe-dev               | llvm-project/clang |
-| cfe-users             | **TODO** llvm-project/clang or create new category? |
-| clangd-dev            | llvm-project/clangd |
-| devmtg-organizers     | **TODO** create new category |
+| cfe-dev               | Clang Compiler & Tools |
+| cfe-users             | Clang Compiler & Tools/Using Clang |
+| clangd-dev            | Clang Compiler & Tools/clangd |
+| devmtg-organizers     | Deprecated |
 | Docs                  | infrastructure/documentation |
-| eurollvm-organizers   | **TODO** create new category |
+| eurollvm-organizers   | Deprecated |
 | flang-commits         | no migration at the moment |
-| flang-dev             | llvm-project/flang |
+| flang-dev             | LLVM Based Subprojects/flang |
 | gsoc                  | community/gsoc |
 | libc-commits          | no migration at the moment |
-| libc-dev              | llvm-project/libc |
-| Libclc-dev            | **TODO** create new category |
+| libc-dev              | Runtimes/libc |
+| Libclc-dev            | Runtimes/libclc |
 | libcxx-bugs           | no migration at the moment |
 | libcxx-commits        | no migration at the moment |
-| libcxx-dev            | llvm-project/libcxx |
+| libcxx-dev            | Runtimes/libcxx |
 | lldb-commits          | no migration at the moment |
-| lldb-dev              | llvm-project/lldb |
-| llvm-admin            | **TODO** create new category |
-| llvm-announce         | **TODO** create new category |
+| lldb-dev              | LLVM Based Subprojects/lldb |
+| llvm-admin            | Replace with bugzilla or Github Issues |
+| llvm-announce         | Announce |
 | llvm-branch-commits   | no migration at the moment |
 | llvm-bugs             | no migration at the moment |
 | llvm-commits          | no migration at the moment |
-| llvm-dev              | llvm-project/llvm |
+| llvm-dev              | LLVM Dev List Archives |
 | llvm-devmeeting       | community/devmtg |
-| llvm-foundation       | **TODO**  create new category|
+| llvm-foundation       | community/foundation |
 | Mlir-commits          | no migration at the moment |
 | Openmp-commits        | no migration at the moment |
-| Openmp-dev            | llvm-project/openmp |
+| Openmp-dev            | Runtimes/openmp |
 | Parallel_libs-commits | no migration at the moment |
-| Parallel_libs-dev     | **TODO** use llvm-project/pstl or create new category ? |
-| Release-testers       | **TODO** create new category |
-| Test-list             | **TODO** create new category |
-| vmkit-commits         | no migration at the moment |
+| Parallel_libs-dev     | Runtimes/libcxx |
+| Release-testers       | community/releases |
+| Test-list             | Deprecated |
+| vmkit-commits         | Deprecated |
 | WiCT                  | community/women-in-compilers-and-tools/ |
```

Comment:
This is using the URL format instead of the name "Community / Women in Compilers and Tools".  It would be good to be consistent one way or the other.

Reply:
Agreed. I think without the dashes.